tests/unit/test_openapi_to_yaml.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+import pytest
+
+from scanapi.errors import EmptyOpenAPIFileError, InvalidOpenAPIFileError
+from scanapi.openapi_to_yaml import openapi_to_yaml
+
+
+class TestOpenAPIToYAML:
+    class TestOpenAPIToYAMLFunction:
+        class TestWhenProvidedWithCorrectFile:
+            def test_should_create_api_yaml_file(self):
+                openapi_to_yaml('../data/openapi.json')
+
+                assert os.path.exists('./api.yaml')
+
```

Comment:
Can we have an assert here for the expected result? We could create a file in the data folder with the result:

```yaml
api:
    endpoints:
    -   name: Uber API
        path: https://api.uber.com/v1
        requests:
        -   name: Product Types
            path: /products
            method: get
        -   name: Price Estimates
            path: /estimates/price
            method: get
        -   name: Time Estimates
            path: /estimates/time
            method: get
        -   name: User Profile
            path: /me
            method: get
        -   name: User Activity
            path: /history
            method: get
```

and assert the content matches. What do you think?

Reply:
@abreumatheus I was noticing that the query params, for example, are not is the expected result. Is this intentional? Does it work for headers, body and others http methods than `get` too? If not, we can create more issues for them, no problem, I just want to understand the context here. Thanks!!

Reply:
@camilamaia Will assert the expected result. I really forgot about the query/path parameters and headers hahaha. I was analyzing it here and i have to work on a couple more things too, like ofering support for the new 3.0 version of OpenAPI that, among some differences, can also come on YAML format now. I was thinking about closing this PR, since it will require a lot of new things and open another one when it's all trully ready and beautiful 😄 . What do you think?

Reply:
@abreumatheus I added you to the team which can push stuff, sorry! I needed you to accept the invite first. But it should be all set now 👌  Please, let me know if you keep having problems.

What you can do is to work in a branch as a draft. Because this is getting in a good shape. Actually, the way you prefer works for me!

Good catch about the OpenAPI 3.0 👀 I was forgetting it too!